scripts/cli_options.py

```diff
@@ -210,7 +210,7 @@ def check_regtest(blockchain_start=True):
         #to allow testing of confirm/unconfirm callback for multiple txs
         jm_single().bc_interface.tick_forward_chain_interval = 10
         jm_single().bc_interface.simulating = True
-        jm_single().maker_timeout_sec = 15
+        jm_single().maker_timeout_sec = 30
```

Comment:
Do you definitely still need this? I'd really prefer 15 seconds for testing tumbler runs, so it gets done in a couple of minutes or so.

Reply:
15 seconds doesn't work at all on my machine (possibly because it has a hard drive rather than an SSD). But I agree that change perhaps doesn't belong in this PR.

Reply:
Right. Feel free to keep it then.